src/Driver/solver_configs.jl

```diff
@@ -180,18 +182,18 @@ function SolverConfiguration(
             numerical_flux_first_order,
             numerical_flux_second_order,
             numerical_flux_gradient,
-            state_auxiliary = dg.state_auxiliary,
+            state_auxiliary = state_auxiliary,
         )
         slow_model = RemainderModel(bl, (linmodel,))
-        slow_dg = DGModel(
+        dg = DGModel(
             slow_model,
             grid,
             numerical_flux_first_order,
             numerical_flux_second_order,
             numerical_flux_gradient,
-            state_auxiliary = dg.state_auxiliary,
+            state_auxiliary = state_auxiliary,
         )
```

Comment:
Would this work?

```julia
dg = DGModel(
    balance_law,
    grid,
    numerical_flux_first_order,
    numerical_flux_second_order,
    numerical_flux_gradient,
    state_auxiliary = state_auxiliary,
)
slow dg = DGModel(
    slow_model,
    grid,
    numerical_flux_first_order,
    numerical_flux_second_order,
    numerical_flux_gradient,
    state_auxiliary = dg.state_auxiliary,
    state_gradient_flux = dg.state_gradient_flux,
    states_higher_order = dg.states_higher_order,
)
```

Reply:
That would be much neater!

Reply:
It appears that `fast_dg` is also using `dg.state_auxiliary`. Should these parameters (for `state_gradient_flux` and `states_higher_order` be added there as well? And if so, then should they also be applied to the vertical `DGModel` created for IMEX solvers?

Reply:
possibly, though in current use cases the fast (linear) models do not have diffusion so it should have no impact.